glances/docker-compose.instance.yaml

```diff
@@ -37,6 +38,13 @@ services:
               capabilities: [gpu]
     #@ end
     
+    volumes:
+      #@ if container_metrics:
+      - /var/run/docker.sock:/var/run/docker.sock:ro
+      #@ end
+      #! The following line causes Glances to display host's OS detail instead of container's
+      - /etc/os-release:/etc/os-release:ro
```

Comment:
Will you also test if you can do the timezone and localtime as shown in #285 and then you probably dont need the TZ env var at all.

Reply:
Didn't work. We decided to approve this PR using $TZ.